gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -52,8 +57,18 @@
 
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
-  private static final String GCD = "gcd-head";
-  private static final String GCD_LOC = '/' + GCD + ".zip";
+  private static final String GCD_VERSION = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
Isnt this name a bit misleading? the version is `v1beta2...` no?

Do we ever use either of these pieces individually? why not just store the `GCD_FILENAME` param?